### ISSUE TITLE
PHP 8 - namespaces

### ISSUE THREAD

**Issue by Miosame** [DirectoryLister/DirectoryLister#539]
**Directory Lister Version**

3.6.1

**PHP Version**

8.0.0

**PHP Extensions**

```
[PHP Modules]
calendar
Core
ctype
date
exif
FFI
fileinfo
filter
ftp
gettext
hash
iconv
json
libxml
openssl
pcntl
pcre
PDO
Phar
posix
readline
Reflection
session
shmop
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
Zend OPcache
zlib

[Zend Modules]
Zend OPcache
```

**HTTP Server**

php serving itself

**Describe the bug**

`syntax error, unexpected namespaced name "HttpMiddlewares\Expires", expecting "]" in /DirectoryLister-3.6.1/app/cache/CompiledContainer.php on line 227`

That's this snippet, specifically the `new \ContainerInterface HttpMiddlewares\Expires($container->get('http_expires')),` line.

```php
    protected function get11()
    {
        return $this->resolveFactory(static function (\ContainerInterface $container): array {
        return [
            Middlewares\WhoopsMiddleware::class,
            new \ContainerInterface HttpMiddlewares\Expires($container->get('http_expires')),
        ];
    }, 'middlewares');
    }
```

**Steps To Reproduce**

serve the project and every hit on / throws that (fatal-) error

**Additional context**

![image](https://user-images.githubusercontent.com/8201077/102655139-14719f80-4172-11eb-9ec0-657f87df06cb.png)

PHP 8 is [now stable](https://www.php.net/releases/8.0/en.php) :+1:

**Comment by PHLAK** [comment]
Thanks, I've been meaning to find some time to get the next release ready and wanted to verify PHP 8 support with that. I'll be sure to look into this when I do.

**Comment by PHLAK** [comment]
Resolved with [v3.7.0](https://github.com/DirectoryLister/DirectoryLister/releases/tag/3.7.0)
